src/test/pow_tests.cpp

```diff
@@ -48,12 +50,13 @@ BOOST_AUTO_TEST_CASE(get_next_work_lower_limit_actual)
     SelectParams(CBaseChainParams::MAIN);
     const Consensus::Params& params = Params().GetConsensus();
 
-    int64_t nLastRetargetTime = 1279295950; // NOTE: Not an actual block time
+    int64_t nLastRetargetTime = 1279296753; // NOTE: Not an actual block time
     CBlockIndex pindexLast;
     pindexLast.nHeight = 68543;
     pindexLast.nTime = 1279297671;  // Block #68543 of Bitcoin
+    // This represents an average difficulty in the current algorithm
```

Comment:
I think they can be; this one for obvious reasons, and `0x200f0f0f` in block 17 (when the previous blocks have all been min-difficulty due to not having enough blocks to fill the averaging window).